app/dao/notifications_dao.py

```diff
@@ -335,31 +334,33 @@ def delete_notifications_older_than_retention_by_type(notification_type, qry_lim
 @statsd(namespace="dao")
 @transactional
 def insert_notification_history_delete_notifications(
-    notification_type, service_id, start_time, end_time, qry_limit=10000
+    notification_type, service_id, timestamp_to_delete_backwards_from, qry_limit=50000
 ):
+    # Setting default query limit to 50,000 which take about 48 seconds on current table size
+    # 10, 000 took 11s and 100,000 took 1 min 30 seconds.
     drop_table_if_exists = """
         DROP TABLE if exists NOTIFICATION_ARCHIVE
         """
     select_into_temp_table = """
          CREATE TEMP TABLE NOTIFICATION_ARCHIVE AS
-         SELECT id
+         SELECT id, job_id, job_row_number, service_id, template_id, template_version, api_key_id,
+             key_type, notification_type, created_at, sent_at, sent_by, updated_at, reference, billable_units,
+             client_reference, international, phone_prefix, rate_multiplier, notification_status,
+              created_by_id, postage, document_download_count
```

Comment:
Optional: If we were to add a new column to notification history we would always need to remember it also add it here. Would be nice to have this happen automatically but think it's not essential for this PR

Reply:
Yes - there is a way to just use the list of columns from a table. This code can be refactored later for things like that.

Reply:
Yes - there is a way to just use the list of columns from a table. This code can be refactored later for things like that.